src/plugin.ts

```diff
@@ -145,27 +143,6 @@ export default (opts: PluginOptions = {}): Plugin => {
         }
       }
 
-      if (opts.svgo !== false)
-        this.transform = async function (code, id) {
```

Comment:
This `transform` hook is for `.svg` files imported by JavaScript modules, so we shouldn't remove it.